pkg/build/graph/analysis/bc.go

```diff
@@ -4,30 +4,38 @@ import (
 	"fmt"
 	"sort"
 	"strings"
+	"time"
 
 	"github.com/gonum/graph"
 	"github.com/gonum/graph/topo"
 
+	"k8s.io/kubernetes/pkg/api/unversioned"
+
 	osgraph "github.com/openshift/origin/pkg/api/graph"
 	buildapi "github.com/openshift/origin/pkg/build/api"
 	buildedges "github.com/openshift/origin/pkg/build/graph"
 	buildgraph "github.com/openshift/origin/pkg/build/graph/nodes"
+	imageapi "github.com/openshift/origin/pkg/image/api"
 	imageedges "github.com/openshift/origin/pkg/image/graph"
 	imagegraph "github.com/openshift/origin/pkg/image/graph/nodes"
 )
 
 const (
-	TagNotAvailableWarning     = "ImageStreamTagNotAvailable"
-	LatestBuildFailedErr       = "LatestBuildFailed"
-	MissingRequiredRegistryErr = "MissingRequiredRegistry"
-	MissingImageStreamErr      = "MissingImageStream"
-	CyclicBuildConfigWarning   = "CyclicBuildConfig"
+	TagNotAvailableWarning         = "ImageStreamTagNotAvailable"
+	LatestBuildFailedErr           = "LatestBuildFailed"
+	MissingRequiredRegistryErr     = "MissingRequiredRegistry"
+	MissingOutputImageStreamErr    = "MissingOutputImageStream"
+	CyclicBuildConfigWarning       = "CyclicBuildConfig"
+	MissingImageStreamTagWarning   = "MissingImageStreamTag"
```

Comment:
> So let me reword, see if I have what you are saying: If there are BuildConfigs with outputs that match the input istag, we call it a warning, but if there are none, we call it an error.
> 
> Is that what you intended?

I agree with @kargakis, but if you want to make it a follow-up issue I'm ok taking this without the update.  This is clearly better than what we have now.